tox.ini

```diff
@@ -0,0 +1,42 @@
+[tox]
+envlist =
+    py27-sentry811
+    py27-sentry812
+    py27-sentry813
+    py27-sentry814
+    py27-sentrylatest
+    linting
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements_base.txt
+    sentry811: sentry==8.11.*
+    sentry811: sentry-plugins==8.11.*
+    sentry812: sentry==8.12.*
+    sentry812: sentry-plugins==8.12.*
+    sentry813: sentry==8.13.*
+    sentry813: sentry-plugins==8.13.*
+    sentry814: sentry==8.14.*
+    sentry814: sentry-plugins==8.14.*
+    sentrylatest: sentry
+    sentrylatest: sentry-plugins
+commands = py.test -vvv {posargs:tests}
+
+[testenv:linting]
+deps =
+    flake8
+    # pygments required by rst-lint
+    pygments
+    restructuredtext_lint
+commands =
+    flake8 sentry_zendesk tests
+    rst-lint CONTRIBUTING.rst HISTORY.rst README.rst
+
+# Run on a separate env and job so we can run the tests in develop mode, which
+# makes it easier to run coverage and specify the sources.
+[testenv:coverage]
```

Comment:
Well, I got that from [pytest](https://github.com/pytest-dev/pytest/blob/master/.travis.yml#L11), but now I realized that pytest excludes it from ```envlist``` because the coverage date upload happens when running ```tox```, so someone could accidentally upload data from a local machine. As here we are uploading the data as a travis step, I'll add it to the ```envlist```.